x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_exceptions/api/create_rule_exceptions/route.ts

```diff
@@ -269,20 +277,24 @@ export const createAndAssociateDefaultExceptionList = async ({
     ? existingRuleExceptionLists.filter((list) => list.type !== ExceptionListTypeEnum.RULE_DEFAULT)
     : existingRuleExceptionLists;
 
-  await detectionRulesClient.patchRule({
-    rulePatch: {
-      rule_id: rule.params.ruleId,
-      ...rule.params,
-      exceptions_list: [
-        ...ruleExceptionLists,
-        {
-          id: exceptionListToAssociate.id,
-          list_id: exceptionListToAssociate.list_id,
-          type: exceptionListToAssociate.type,
-          namespace_type: exceptionListToAssociate.namespace_type,
-        },
-      ],
-    },
+  // if patchRule fails, delete the exception list created above
```

Comment:
Could you elaborate on this comment? Is it still relevant? According to the diff `patchRule` has been removed.